src/datasource/OGC.js

```diff
@@ -162,6 +162,7 @@ export const WMSInfoFormat = {
  * @typedef {Object} WMSLayer
  * @property {string} name The layer name (WMS) and/or feature type name (WFS)
  * @property {boolean} [queryable] Whether the the layer is queryable or not. Defaults to `false`.
+ * @property {boolean} [getData]
```

Comment:
no description ?

Reply:
Effectively I just see that's used, and I don't really know what is it...
I can write a Java comment like « Be able to get the data » But is just mask that I don't know what's for... Any ides about that?

Reply:
Looks redundant to me but it looks to be a possibility to query (like queryable...). See: https://github.com/camptocamp/ngeo/blob/1f85929ca55c657f0405bf61deb0942d7221dce4/contribs/gmf/src/themes.js#L154-L155

Reply:
Excellent, thanks :-)